test/parallel/test-crypto.js

```diff
@@ -300,3 +300,9 @@ testEncoding({
 testEncoding({
   defaultEncoding: 'latin1'
 }, assertionHashLatin1);
+
+// isKeyObject checks
+assert(crypto.isKeyObject(crypto.createSecretKey(Buffer.from('foo'))));
+const keypair = crypto.generateKeyPairSync('ec', { namedCurve: 'P-256' });
+assert(crypto.isKeyObject(keypair.privateKey));
+assert(crypto.isKeyObject(keypair.publicKey));
```

Comment:
Can you please add negative test cases? So far, the test would pass even if `isKeyObject` was `() => true`. Checking invalid types (`null`, `undefined`, `false`, `5`, `foo`) and invalid objects (`Buffer`) in a loop might be a good idea.